src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -5498,6 +5619,108 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/sites/{siteId}/groups/{groupId}':
```

Comment:
I would suggest we use "group-members" instead of "group" to make it clearer we are talking about the groups that are members of the site and keep it consistent with the other member endpoints.